packages/docusaurus-theme-classic/src/__tests__/__snapshots__/translations.test.ts.snap

```diff
@@ -16,6 +16,10 @@ exports[`getTranslationFiles returns translation files matching snapshot 1`] = `
         "description": "Navbar item with label Dropdown item 2",
         "message": "Dropdown item 2",
       },
+      "logo.alt": {
+        "description": "The alt text of navbar logo",
+        "message": "navbar alt logo",
```

Comment:
All of these would make more sense readability wise if they were in the form "<thing> <component> alt text" rather than "alt <component>" which could be interpreted as "alternative logo".